store/mockstore/unistore/cophandler/cop_handler.go

```diff
@@ -436,12 +436,15 @@ func buildRespWithMPPExec(chunks []tipb.Chunk, counts, ndvs []int64, exec mppExe
 	resp.ExecDetailsV2 = &kvrpcpb.ExecDetailsV2{
 		TimeDetail: resp.ExecDetails.TimeDetail,
 	}
-	data, err := proto.Marshal(selResp)
-	if err != nil {
-		resp.OtherError = err.Error()
+	data, mErr := proto.Marshal(selResp)
+	if mErr != nil {
+		resp.OtherError = mErr.Error()
 		return resp
 	}
 	resp.Data = data
+	if err != nil {
+		resp.OtherError = err.Error()
```

Comment:
Seems this line failed the tests.

Reply:
The calculation error should not be reported in the `other` error field, here unistore is not compatible with the tikv logic.